xbuild/ios-target/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "ios-xbuild-target"
```

Comment:
Nit: given the directory structure I'd name these `xbuild-ios-target`. After all, `xbuild` is targeting an architecture, the architecture isn't targeting `xbuild`.